gandalf/build.gradle

```diff
@@ -65,9 +65,9 @@ gnag {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
-    compile 'com.android.support:appcompat-v7:23.2.0'
-    compile 'com.squareup.okhttp3:okhttp:3.0.1'
-    compile 'com.google.code.gson:gson:2.5'
+    compile 'com.android.support:appcompat-v7:25.2.0'
+    compile 'com.squareup.okhttp3:okhttp:3.6.0'
+    compile 'com.google.code.gson:gson:2.8.0'
 
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
```

Comment:
Don't you have the dependency updates plugin set up? Recommended if not. https://github.com/ben-manes/gradle-versions-plugin